.autocorrectignore

```diff
@@ -1 +1,5 @@
-files/zh-cn/conflicting
\ No newline at end of file
+files/
+docs/
+!docs/zh-cn/
+!files/zh-cn/
+files/zh-cn/web/javascript/reference/global_objects/bigint/tolocalestring/index.md
```

Comment:
Disable this special file via:

https://github.com/mdn/translated-content/pull/8781#discussion_r982369678

Reply:
https://github.com/mdn/translated-content/pull/8781#discussion_r985033499

Reply:
@yin1999 Is this a blocker? If so, what needs to happen before this PR can be merged?

Reply:
The AutoCorrect `textRules` new features can handle this.

https://github.com/huacnlee/autocorrect/releases/tag/v2.2.0

Can we add some special rules?

```
textRules:
  9.9亿: 0
  一二三,四五六.七八九: 0
  一二三,四五六,七八九,一二三,四五六,七八九: 0
```

Reply:
Sounds better, what's your opinion @caugner

## background

In Chinese, we use `，` instead of `,` between short Chinese sentences, but the sentence metioned above is the output of JS API (See: <https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString#using_locales>)

![image](https://user-images.githubusercontent.com/15844309/197668291-09ffd3f2-4021-497a-a06c-99eb862bfd75.png)

This tool will try to "correct" it.

Reply:
Sounds good to me, even though you will need to always bear this in mind when adding new code examples. Then again, you probably don't add examples with commas in their output everyday. 🙂